plugins/mcreator-localization/help/default/armor/max_damage_absorbed.md

```diff
@@ -1,14 +1,14 @@
 This parameter defines armor durability and is effectively applied as:
 
-* helmet: max_damage_absorbed * 13
-* chestplate: max_damage_absorbed * 15
-* leggings: max_damage_absorbed * 16
-* boots:  max_damage_absorbed * 11
+* helmet: max_damage_absorbed * 11
+* chestplate: max_damage_absorbed * 16
+* leggings: max_damage_absorbed * 15
+* boots:  max_damage_absorbed * 13
 
 Vanilla armor uses the following factors:
 
 * Leather armor: 5
-* Chainmail armor: 15
+* Chainmail/Iron armor: 15

```

Comment:
You should keep them split as we don't group like this in other tips

Reply:
Ok

Reply:
Done